llvm_objects.py

```diff
@@ -162,7 +205,7 @@ def _get_instructions(self, l_block, w_module):
             prev = current_instr
             w_instrs.append(current_instr)
             l_instr = llwrap.LLVMGetNextInstruction(l_instr)
-        return w_instrs
+        return w_instrs[:]
```

Comment:
The list should never mutate. I'm returning a copy of the list because the return value of this method gets assigned to an immutable attribute. If I had returned the list itself, then the attribute wouldn't have been immutable.